proxy-server/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+
+  "manifest_version": 2,
+  "name": "Proxy Server",
+  "description": "Uses the proxy API to send all requests through a proxy server, optionally with authentication. Stores settings using the storage API.",
+  "version": "1.0",
+
+  "icons": {
+    "48": "icons/block.svg",
+    "96": "icons/block.svg"
```

Comment:
Can you please suggest where to get these? I found something relevant: https://thenounproject.com/search/?q=proxy&i=831371 but then I noticed the license.